cmd/dra-example-webhook/main.go

```diff
@@ -0,0 +1,304 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
```

Comment:
For context, this webhook implementation is largely based on the [agnhost one](https://github.com/kubernetes/kubernetes/tree/9e0353b81e1c4bd437431b689624494ef71f8038/test/images/agnhost/webhook).